Sprint-Mission1/js/login.js

```diff
@@ -0,0 +1,96 @@
+document.addEventListener("DOMContentLoaded", function () {
+  const emailInput = document.getElementById("pandaEmail");
+  const passwordInput = document.getElementById("pandaPassword");
+  const loginButton = document.getElementById("loginButton");
+  const emailError = document.getElementById("e-error");
+  const passwordError = document.getElementById("p-error");
+  const closeEyesbtn = document.querySelector(".close-eyes");
+  const openEyesbtn = document.querySelector(".open-eyes");
+
+
+
+  // 이메일 유효성 검사
+  function validateEmail() {
+    const emailValue = emailInput.value.trim();
+    emailError.textContent = "";
+    emailInput.classList.remove("error");
+
+    if (!emailValue) {
+      emailError.textContent = "이메일을 입력해주세요.";
+      emailInput.classList.add("error");
+      return false;
+    } else if (!/^[^\s@]+@[^\s@]+\.[^\s@]+$/.test(emailValue)) {
+      emailError.textContent = "잘못된 이메일 형식입니다.";
+      emailInput.classList.add("error");
+      return false;
+    }
+    return true;
+  }
+
+  // 비밀번호 유효성 검사
+  function validatePassword() {
+    const passwordValue = passwordInput.value.trim();
+    passwordError.textContent = "";
+    passwordInput.classList.remove("error");
+
+    if (!passwordValue) {
+      passwordError.textContent = "비밀번호를 입력해주세요.";
+      passwordInput.classList.add("error");
+      return false;
+    } else if (passwordValue.length < 8) {
+      passwordError.textContent = "비밀번호를 8자 이상 입력해주세요.";
+      passwordInput.classList.add("error");
+      return false;
+    }
+    return true;
+  }
+  // 비밀번호 숨김 상태 확인 함수
+  function togglePasswordVisibility(passwordType, passclosebtn, passopenbtn) {
```

Comment:
지금도 잘 작성해주셨지만, 아래와 같이 짧게 줄여쓸 수도 있습니다ㅎㅎ
```suggestion
function togglePasswordVisibility(passwordType, passclosebtn, passopenbtn) {
  const isPassword = passwordType.getAttribute('type') === 'password';
  passwordType.setAttribute('type', isPassword ? 'text' : 'password');
  passclosebtn.classList.toggle('visible', !isPassword);
  passopenbtn.classList.toggle('visible', isPassword);
}
```

Reply:
그리고 변수명에 약어를 쓰시는건 큰 문제는 되지 않지만, 저는 개인적으로 지양하는 편인데요.
개발자마다 단어를 줄이는 방식이 다르기 때문에 헷갈릴 수 있기 때문이에요.
특히 여기서 password 를 pass 로 줄이셨는데, 동사 pass 와도 혼동될 수 있기때문에 차라리 pw 로 줄이시는게 나을 것 같아요.

저라면 passwordVisibleButton / passwordInvisibleButton 이렇게 작성할 것 같긴하네요 ㅎㅎ